gulp/sass.js

```diff
@@ -70,8 +69,9 @@ module.exports = (gulp, plugins, blueprint) => {
                     );
                 },
             }),
-            autoprefixer(config.autoprefixer),
-        ];
+        ] : [];
+        // always run autoprefixer
+        postcssPlugins.push(autoprefixer(config.autoprefixer));
```

Comment:
yeah i considered that but decided to do this to avoid repetition. i also like the empty array for the `false` case. very easy to reason about.